examples/map/compare_rotation_results.py

```diff
@@ -0,0 +1,47 @@
+# # Comparing Results of `map.rotate` Between Library Implementations
```

Comment:
The formatting of this example is a bit off, in particular
- Needs a short title
- Needs a description after the title
- The top comment should be formatted with a multiline comment, ie. `"""` 
- The comments below should have lines of `#` to create sections

Looking at other examples should help, but do ask if any of that doesn't make sense!

Reply:
Ok, I have modified the formatting according to the other examples.